packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -924,7 +924,14 @@ export class ContainerRuntime
 	 * @param callback - the callback to be invoked
 	 */
 	public ensureNoDataModelChanges<T>(callback: () => T): T {
-		this.ensureNoDataModelChangesCalls++;
+		if (++this.ensureNoDataModelChangesCalls === 100) {
```

Comment:
Curious if this gives any better debuggability than whatever JS runtime provides RE exhausting stack. If we are not gaining much then I'd probably not add it.

Reply:
I think there may be a race between how fast we exhaust the stack vs some throttling or other limitations we might find when we send the ops. With flushing explicitly blocked, the latter is unlikely to happen, but I don't believe we should allow this scenario as it's a sign of a bug. I'll experiment with what's happening if we don't do this at all.

Reply:
I think we can remove this. The JS runtime will throw. As long as we can't flush during this time, it's ok.